### DISCUSSION TITLE
Composters

### DISCUSSION THREAD

**Discussion by ghost** [Roadhog360/Et-Futurum-Requiem#116] [Ideas]
As someone who really likes to farm, this would help out a lot! Composters are very useful for farming. Ever since they added composters in the game, I've felt like I've appreciated it's mechanic as I used it over time. The idea of composting plants and seeds into bone meal is so simple but so ideal. I use them a lot with major farms, using an automatic compost hopper system.

I understand if this isn't possible. I'd be okay with either the old crafting recipe (4 fences with 3 wooden planks?) or the new recipe (7 wooden slabs, similar to a cauldron recipe)

Was told that the GardenStuff mod has a composter- but after using it, it's just so slow and inefficient. Hence, wondering if backporting the vanilla composter in this mod would be cool.

**Comment by Roadhog360** [comment]
Composters are already in the latest builds, which have not yet been released.

**Comment by Roadhog360** [comment]
Composters have been added for a while now, closing
